portfolio/src/main/java/com/google/sps/comment/repository/CommentRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.comment.repository;
+
+import com.google.sps.comment.Comment;
+import org.jetbrains.annotations.Nullable;
+import java.util.List; 
+
+public interface CommentRepository{
+    public void saveComment(Comment comment);
+
+    public List<Comment> getAllComments();
+
+    public List<Comment> getGivenNumberOfComments(int maxNoComments);
```

Comment:
add doc for non-straight-forward public methods

Reply:
done